App.js

```diff
@@ -259,19 +265,25 @@ class App extends React.Component<Props, *> {
                 theme={current === LIGHT_THEME ? 'light' : 'dark'}
                 language={i18n.language}
               />
-              {!!SHOW_THEME_TOGGLE &&
+              {!!getEnv().SHOW_THEME_TOGGLE &&
               <Button
                 title={`THEME: ${current}`}
                 onPress={() => {
                   const themeToChangeTo = current === LIGHT_THEME ? DARK_THEME : LIGHT_THEME;
                   setAppTheme(themeToChangeTo);
                 }}
               />}
-              {!!SHOW_LANG_TOGGLE && <Button
+              {!!getEnv().SHOW_LANG_TOGGLE && <Button
                 title={`Change lang (current: ${i18n.language})`}
                 onPress={() => changeLanguage(i18n.language === 'fr' ? 'en' : 'fr')}
               />}
               {!!activeWalkthroughSteps.length && <Walkthrough steps={activeWalkthroughSteps} />}
+              {this.state.env === 'staging' &&
```

Comment:
Just thinking – can we have `staging` and `production` as constants as well as we type those in few different places, wdyt?